Source/WebCore/Modules/identity/CredentialRequestCoordinator.h

```diff
@@ -53,12 +53,12 @@ class CredentialRequestCoordinator final : public RefCounted<CredentialRequestCo
 
 public:
     static Ref<CredentialRequestCoordinator> create(UniqueRef<CredentialRequestCoordinatorClient>&&, Page&);
-    WEBCORE_EXPORT bool presentPicker(CredentialPromise&&, DigitalCredentialsRequestData&&, RefPtr<AbortSignal>);
+    WEBCORE_EXPORT void presentPicker(CredentialPromise&&, DigitalCredentialsRequestData&&, RefPtr<AbortSignal>);
     WEBCORE_EXPORT void abortPicker(JSC::JSValue reason);
-
     ~CredentialRequestCoordinator();
 
 private:
+    static bool canPresentDigitalCredentialsUI() { return m_hasDigitalCredentialsUI; }
```

Comment:
Seems like an extra indirection, why not just define the static variable like this?

Also, you can make it `constexpr`!

```suggestion
    static constexpr bool canPresentDigitalCredentialsUI {
#if HAVE(DIGITAL_CREDENTIALS_UI)
        true
#else
        false
    };
```

And then you can use it like such:

```cpp
void CredentialRequestCoordinator::foobar()
{
    if (canPresentDigitalCredentialsUI) {
        ...
    }
}
```